docs/Hosts/Spawn-Hosts.md

```diff
@@ -114,23 +93,15 @@ The example sleep schedule above will automatically turn the host on from 8am to
 The host will be automatically turned off overnight from 8pm to 8am on Monday to Friday, and will be off for the entire
 weekend.
 
-From the create/edit host modal, you can pick which days you'd like the host to be on, and the time that you want your
-host to be on for those days. The time zone for the host sleep schedule will be based on the time zone you choose in
-your [Spruce preferences](https://spruce.mongodb.com/preferences/profile). You can edit this schedule whenever you want.
-
-When setting a sleep schedule in the UI, you also have the option to participate in the beta test. If you opt into the
-beta test, the sleep schedule that you set will take effect on your host, stopping and starting your host according to
-the schedule you configure; if you do not opt in, your sleep schedule will be set but will have no effect on your host
-during the beta test period (i.e. until September 3). You can opt in or out freely during the beta test for this
-feature. **It's highly recommend that when you set a sleep schedule for your unexpirable host(s), you also opt into the
-beta test to verify that your sleep schedule is working the way you want.**
+From the create/edit host modal, you can pick which days you'd like the host to be on, the time that you want your
+host to be on for those days, and the time zone of the schedule. You can edit this schedule whenever you want.
```

Comment:
it would be nice to include a screenshot here if you feel like it

Reply:
The screenhot is above (L90), this is just explaining the options in the screenshot.